structs.go

```diff
@@ -582,37 +582,79 @@ type VoiceState struct {
 
 // A Presence stores the online, offline, or idle and game status of Guild members.
 type Presence struct {
-	User   *User    `json:"user"`
-	Status Status   `json:"status"`
-	Game   *Game    `json:"game"`
-	Nick   string   `json:"nick"`
-	Roles  []string `json:"roles"`
-	Since  *int     `json:"since"`
+	// The user the presence updated for.
+	User *User `json:"user"`
+
+	// Roles the user is in.
+	Roles []string `json:"roles"`
+
+	// The status of the user.
+	Status Status `json:"status"`
+
+	// Either nil or the user's current activity.
+	Game *Activity `json:"game"`
+
+	// The user's platform-dependent status.
+	ClientStatus *ClientStatus `json:"client_status"`
+
+	// User's current activities.
+	Activities []*Activity `json:"activities"`
 }
 
-// GameType is the type of "game" (see GameType* consts) in the Game struct
-type GameType int
+// ActivityType is the type of "activity" (see ActivityType* consts) in the Activity struct
+type ActivityType int
 
 // Valid GameType values
 const (
-	GameTypeGame GameType = iota
-	GameTypeStreaming
-	GameTypeListening
-	GameTypeWatching
+	ActivityTypeGame ActivityType = iota
+	ActivityTypeStreaming
+	ActivityTypeListening
 )
 
-// A Game struct holds the name of the "playing .." game for a user
-type Game struct {
```

Comment:
The struct is called Game to reflect the same name as the api

Reply:
As I wasn't aware breaking changes weren't allowed, I changed it to _Activity_ as the json field is still called _game_ but the type is referred to as an _activity object_. But as said earlier will change this back to _Game_.

Reply:
We don't allow breaking changes to make sure people can always update the lib to the latest version. Unless we release a new major version, in which case breaking changes are allowed